Qt/mainwindow.h

```diff
@@ -122,28 +132,18 @@ private slots:
 	// Sound
 	void audioAct() { g_Config.bEnableSound = !g_Config.bEnableSound; }
 
+	// Cheats
+	void cheatsAct() { g_Config.bEnableCheats = !g_Config.bEnableCheats; }
+
 	void fullscrAct();
 	void raiseTopMost();
 	void statsAct() {
 		g_Config.bShowDebugStats = !g_Config.bShowDebugStats;
 		NativeMessageReceived("clear jit", "");
 	}
+	void VSyncAct() { g_Config.bVSync = !g_Config.bVSync; }
```

Comment:
As mentioned, this ini setting doesn't do anything without platform hookup.

This implies it can be done using a surface format setting: https://stackoverflow.com/questions/37319328/qt-5-disable-vsync

There was talk in #9736 and #12024 of potentially making it possible to disable vsync on Linux, but I'd suggest we don't tackle that yet and leave this option out for now.

-[Unknown]